shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/scenario/consistencycheck/ConsistencyCheckJobAPIImpl.java

```diff
@@ -110,10 +110,10 @@ public Map<String, DataConsistencyCheckResult> getLatestDataConsistencyCheckResu
     
     @Override
     public void persistJobItemProgress(final PipelineJobItemContext jobItemContext) {
-        DataCheckJobItemProgress progress = new DataCheckJobItemProgress();
+        ConsistencyCheckJobProgress progress = new ConsistencyCheckJobProgress();
         progress.setStatus(jobItemContext.getStatus());
-        YamlDataCheckJobProgress yamlDataCheckJobProgress = PROGRESS_SWAPPER.swapToYamlConfiguration(progress);
-        PipelineAPIFactory.getGovernanceRepositoryAPI().persistJobItemProgress(jobItemContext.getJobId(), jobItemContext.getShardingItem(), YamlEngine.marshal(yamlDataCheckJobProgress));
+        YamlConsistencyCheckJobProgress yamlConsistencyCheckJobProgress = PROGRESS_SWAPPER.swapToYamlConfiguration(progress);
```

Comment:
`yamlConsistencyCheckJobProgress` could be `yamlJobProgress`.

`progress` could be `jobProgress`.